### PR TITLE
Feature/15 contact information screen

### LINKED ISSUE #15: Add contact information screen design
# REQUIREMENTS
This is the last part for te registrations, here the users need to provide their full name and their phone or cellphone number. This data is not necessary to ingress.

# DESIGN

image

# Acceptance Criteria
List all required acceptance criteria
- [ ] Implementation must match the provided design

### PR DESCRIPTION
**Pull request by OwenRamire**
close #15 
# Description
- Create the "Información de Contacto" screen layout
- Create a new component that stores the instructions of what the user is going to do on the screen 
-  Reuse the EmailPassTxtInput component where the user will type his name and his phone. Also, using props we tell at this component what keyboard we need on each input. 
- Create new navigation until this new screen

## Screenshots 
- Layout of the screen

Screenshot_1602009892

- using the default keyboard on the "Nombre" input
Screenshot_1602009904

- using the phone-pad keyboard on the "Teléfono" input
Screenshot_1602009918

### CODE REVIEW

**Review by xErikPsx — COMMENTED** [review]
Cool design! Approved!

**Review by xErikPsx — DISMISSED** [review]

File: src/components/DescContactInfo.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {View, Text, StyleSheet} from 'react-native';
+
+const DescContactInfo = () => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.textInstructions}>
+        Esta información servirá para ponernos en contacto contigo más rapido
+        cuando sea necesario
```
Comment (fernandoamz) [inline src/components/DescContactInfo.js]: Add a constant  folder into `src` and create a constants file import the text as variable

File: src/navigations/NavigationLogin.js [hunk 2]
```diff
@@ -49,6 +50,17 @@ const NavigationLogin = () => {
         }}
         component={ConfirmEmail}
       />
+      <Stack.Screen
+        name="ContactInformation"
```
Comment (fernandoamz) [inline src/navigations/NavigationLogin.js:54]:
```suggestion
        name='ContactInformation'
```

File: src/screens/ContactInformation.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+  const putInputs = () => {
+    const inputs = titleInput.map((element) => {
+      if (element === 'Nombre') {
+        return (
+          <EmailPassTxtInput
+            key={element}
+            id={element}
+            title={element}
+            isRequired="Recomendado"
```
Comment (fernandoamz) [inline src/screens/ContactInformation.js:30]:
```suggestion
            isRequired='Recomendado'
```

change all double-quotes for simple-quotes on the files

File: src/screens/ContactInformation.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        );
+      }
+    });
+    return inputs;
+  };
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView>
+        <Text style={styles.title}>Información de Contacto</Text>
```
Comment (fernandoamz) [inline src/screens/ContactInformation.js]: Add all the texts as constants

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by fernandoamz — APPROVED** [review]

**Review by luisarmandodv — APPROVED** [review]

**Review by xErikPsx — COMMENTED** [review]
It looks on point!

**Pull request merged**